contracts/position/PositionEventUtils.sol

```diff
@@ -59,7 +59,7 @@ library PositionEventUtils {
         eventData.uintItems.setItem(10, "collateralTokenPrice.max", params.collateralTokenPrice.max);
         eventData.uintItems.setItem(11, "collateralTokenPrice.min", params.collateralTokenPrice.min);
         eventData.uintItems.setItem(12, "sizeDeltaUsd", params.sizeDeltaUsd);
-        eventData.uintItems.setItem(13, "sizeDeltaInTokens", params.sizeDeltaInTokens);
+        eventData.uintItems.setItem(13, "baseSizeDeltaInTokens", params.baseSizeDeltaInTokens);
```

Comment:
i think this part should not be changed, the event keys are used for e.g. analytics, so we should avoid changing the naming here if possible

Reply:
Changed back the event key